app/javascript/gobierto_investments/webapp/components/MapTour.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div class="map-flyto-container">
+    <div class="map-flyto-header">
+      <div class="site_header_image">
+        <a href="#">
```

Comment:
This link is broken and the src of the image is hardcoded. If you don't know how to get this dynamic ask @entantoencuanto